### ISSUE TITLE
Cards with broken images blank other cards in hand.

### ISSUE THREAD

**Issue by anathomical** [mtgred/netrunner#8]
Drew a card with broken image (Akamatsu Mem Chip).  In the grip, there's just a card outline with no image, and in the preview pane there is a broken image indicator.

Further, as play progresses, more and more cards in the hand show as blank (but do preview correctly).  Here's my best attempt to explain behavior.

When older cards (earlier in the array?) are played, and the card with the broken image moves to the left, cards that are shifted into its old position within the array become blank in the Grip row, but can still be viewed in the preview pane.  New cards drawn are shown normally, but when an older card is played, and everything shifts left, the new card will be blanked.

Basically, from what I can tell, once an array position for the Grip row has a blank image, that image stays blank until the array shrinks enough that the blank-image index is deleted.  At that point, things return to normal.

**Comment by anathomical** [comment]
This is still not an urgent problem for fixing, but since I tested this again today I figured I'd note that it's still live.

**Comment by JoelCFC25** [comment]
@anathomical: is this still an issue?

**Comment by anathomical** [comment]
Unknown if this is still an ongoing issue at the code level as, to the best of my knowledge, the underlying bug that caused some cards to be unable to pull the appropriate card images does not happen for any extant cards.  With that in mind, I feel comfortable closing this ticket for now, and we can always re-open it, or a similar ticket, if the issue arises again.

Basically, the circumstances that cause this issue to express no longer seem to arise, so we can probably call it fixed.
